core/src/main/kotlin/com/gitlab/kordlib/core/entity/Guild.kt

```diff
@@ -73,6 +73,29 @@ class Guild(
      */
     val bannerHash: String? get() = data.banner
 
+    /**
+     * The [PremiumTier] from Nitro boosts,
+     */
+    val premiumTier: PremiumTier get() = data.premiumTier
+
+    /**
+     * Maximal amount of server emotes based on [premiumTier].
+     *
+     * [PremiumTier.Unknown] -> [Int.MIN_VALUE]
+     * [PremiumTier.None] -> 50
+     * [PremiumTier.One] -> 100
+     * [PremiumTier.Two] -> 150
+     * [PremiumTier.Three] -> 250
+     */
+    val maxEmotes: Int
```

Comment:
This property and the information it conveys aren't documented in the API anywhere. 

I'm aware it's visible from the client, but this does carry the risk of not in any way being stable. 
Discord has already taken liberties in changing the perks related to boost levels on a whim during the recent pandemic (not in negative ways, mind you), so I'm extra nervous of this becoming outdated *very* quickly, which makes it a lot less useful for users.

Reply:
Thinking about this some more, I'm not certain this is the right place to put this property either. If it is completely dependent on `PremiumTier` and not the `Guild` object, it should be a property of `PremiumTier`.

I'm not keen on using magic numbers for `Unknown` either. If we do not know the value for all tiers, the property should be nullable instead.

Reply:
What about adding it to PremiumTier and adding some annotation on top of it to mark it as not documented by discord?
Or just remove it completely?

Reply:
If we're adding it, it should definitely end up in `PremiumTier`. 

I'm not keen on marking it with any kind of annotation though, API annotated with Kord annotations have a goal of becoming stable in the long run. Something like this can never become stable until Discord documents it as such, so I'm not sure what we would be conveying through it.

Looking further into this, there *is* some form of [official documentation](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-) on the matter, which makes me not completely opposed to adding this anymore. Linking to this in the documentation of the property/enum would go a long way.

I'm not sure how @HopeBaron stands on this issue.

I guess the more general question is if we should aim to provide data that is only present in the client UI. If that data has value for the bot (which I'd argue is the case here), then I guess I should be fine with that, we'll take on the burden of keeping it up to date.

So the recap, I'm okay with adding the property to `PremiumTier` if the link gets added to the official docs.